geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -56,14 +60,52 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
 
   import HBaseFeatureIndex.{DataColumnFamily, DataColumnQualifier}
 
+  private val logger = LoggerFactory.getLogger("org.locationtech.geomesa.hbase.index.HBaseFeatureIndex")
+
   override def configure(sft: SimpleFeatureType, ds: HBaseDataStore): Unit = {
     super.configure(sft, ds)
+
     val name = TableName.valueOf(getTableName(sft.getTypeName, ds))
     val admin = ds.connection.getAdmin
+    val coproUrl: Option[Path] = if(!ds.config.coprocessorUrl.equals(HBaseDataStoreParams.CoprocessorUrl.sample)) {
+        Some(ds.config.coprocessorUrl)
+      } else {
+        GeoMesaSystemProperties.SystemProperty("geomesa.hbase.coprocessor.path", null).option match {
```

Comment:
instead of a match, this could be more succinctly written as `.option.map(new Path(_))`